bigquery/google/cloud/bigquery/client.py

```diff
@@ -1172,10 +1172,10 @@ def query_rows(
 
         :rtype: :class:`~google.api_core.page_iterator.Iterator`
         :returns:
-            Iterator of row data :class:`tuple`s. During each page, the
-            iterator will have the ``total_rows`` attribute set, which counts
-            the total number of rows **in the result set** (this is distinct
-            from the total number of rows in the current page:
+            Iterator of row data :class:`~google.cloud.bigquery.Row`s. During
```

Comment:
Didn't @tswast change the documentation so that objects were documented in the module where they are defined? IOW, this should be `~google.cloud.bigquery.table.Row`

Also, I'm fairly certain this won't render correctly with Sphinx unless there is a separator before the `s` (like a hyphen):

```
:class:`~google.cloud.bigquery.Row`-s
```

Reply:
Fixed the references in https://github.com/GoogleCloudPlatform/google-cloud-python/pull/4291/commits/0a4fb77d565b3edfdfc76e6041a06dad92d13e9f